docs/config/README.md

```diff
@@ -674,6 +675,40 @@ This does not suppress conda's own prompt modifier, you may want to run `conda c
 format = "[$symbol$environment](dimmed green) "
 ```
 
+## Container
+
+The `container` module displays a symbol and container name, if inside a container.
```

Comment:
Do I read the code right, that only podman containers (e.g. ones with `/run/.containerenv` which contain a `image=...`line) get a non-generic name? If so it would be nice to describe that somehow.

(currently trying to figure out how this interacts with https://github.com/89luca89/distrobox)

Reply:
```
~ via 🐹 v1.16.13 via ☕ v11.0.13 via  via 🐪 v5.34.0 via 🐍 v3.10.1 via 🦀 v1.58.0 
❯ toolbox enter 

~ via 🐹 via ☕ via  via 🐪 v5.34.0 via 🐍 v3.10.0 via 🦀 v1.58.0 
[fedora-toolbox:35] ❯ cat /run/.containerenv
engine="podman-3.4.4"
name="fedora-toolbox-35"
id="1e61eb104cd571b8a8b535542cedcdef225a9f53c6ff24e905b3b2ccc98a98a3"
image="registry.fedoraproject.org/fedora-toolbox:35"
imageid="ab8bc106d4a710a7a27c538762864610467b3559f80b413d30e0a1bfcfe272a5"
rootless=1
graphRootMounted=1
```

Reply:
@jankatins ^